packages/react-scripts/config/paths.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
   appPublic: resolveApp('public'),
   appHtml: resolveApp('public/index.html'),
   appIndexJs: resolveApp('src/index.js'),
+  appCssVariables: resolveApp('src/styles/variables.js'),
```

Comment:
One way to document what fixed paths our CRA fork is using and keep them up to date, is to expose them to */build/paths.json*
In webpack configs, it could be something like 
```var GenerateJsonPlugin = require('generate-json-webpack-plugin');```
and in plugin section:
```new GenerateJsonPlugin('paths.json', paths)```

What do you think?